homeassistant/components/tasmota_irhvac/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "domain": "tasmota_irhvac",
+    "name": "Tasmota Irhvac",
+    "documentation": "https://www.home-assistant.io/integrations/tasmota_irhvac",
+    "requirements": [],
+    "dependencies": [
+      "mqtt",
+      "sensor"
```

Comment:
Why is `sensor` as a dependency?

Reply:
I'm not sure if it should be dependency, but in the code, I'm getting temperature sensor data and I'm looking for state change (with a callback) of this sensor .. If it isn't needed I'll remove it but I can't decide by myself. Can You, please, help me about that?

Reply:
For what do you need this temperature sensor exactly?

Reply:
I'm showing its value on the climate card .. This is the actual current temp in the room .. Not the target temp ..

Reply:
Not sure what to do so no chagne ..